src/logic/operate.js

```diff
@@ -0,0 +1,22 @@
+import Big from 'big.js';
+
+const operator = (num1, num2, operand) => {
+  const x = Big(num1);
+  const y = Big(num2);
+
+  if (operand === '+') {
+    return x.plus(y).toString();
+  } if (operand === '-') {
+    return x.minus(y).toString();
+  } if (operand === '÷') {
+    return x.div(y).toString();
+  } if (operand === 'x') {
```

Comment:
You didn't manage DIVISION BY ZERO.........return something like `INFINITY` on screen or some other form of error management

![Screen Shot 2020-04-06 at 6 50 25 PM](https://user-images.githubusercontent.com/40712459/78592514-1d694d80-783d-11ea-9746-82d5f07f1b4a.png)

Reply:
Thanks for this. I will fix this.